src/index.html

```diff
@@ -1,11 +1,24 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Wbpack Exercise</title>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Leaderboard</title>
 </head>
 <body>
-    <h1>Hello webpack!</h1>
+  <h1 class="header">Leaderboard</h1>
+  <main class="container">
+    <div class="score-container">
+      <h3>Recent scores</h3>
+      <button type="submit" id="refresh">Refresh</button>
```

Comment:
- [ ] Great job 👍 . Kindly make sure the `h3` element and `button` are aligned on the same row.

WireFrame Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/181936267-98337010-7e37-4b83-913e-512a2324428e.png) |  ![image](https://user-images.githubusercontent.com/74814780/181936273-21542fe9-7a6c-40cd-b1c9-d41a59b446c0.png)